src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, Arial, sans-serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
```

Comment:
You have added both top and bottom margins to the `.logo` class. According to the checklist, you should be consistent with your vertical margins - add only top or only bottom margin, but not both. This is because vertical margins can be collapsed in some cases, which could lead to unexpected layout results. Consider removing one of the margins.